### DISCUSSION TITLE
🚀 Feature Request: Uptime kuma integration

### DISCUSSION THREAD

**Discussion by jdrozdnovak** [louislam/dockge#168] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

API

### 🔖 Feature description

As uptime kuma is wonderful tool, there could be GET api, that would list all containers and their statuses. This API would be already preset in uptime kuma and generate monitors per container/docker-compose

### ✔️ Solution

This would just create great stack and would eventually solve some basic integration/alerting

### ❓ Alternatives

Create monitors automatically. Or run uptime kuma on same host. But all doubling work/resources

**Comment by brypie** [comment]
I think this is the same as what I was thinking.
Automatically create monitors in uptime-kuma for all containers managed by Dockge?
